### PR TITLE
support for gopherjs

### PR DESCRIPTION
**Pull request by visualfc**
update for support gopherjs ( https://github.com/gopherjs/gopehrjs )

### CODE REVIEW

File: internal/graphicsdriver/opengl/context_js.go [hunk 1]
```diff
@@ -12,6 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build js
+// +build js
```
Comment (hajimehoshi) [inline internal/graphicsdriver/opengl/context_js.go:16]: I don't think we should care this. Wouldn't it be possible for GopherJS to recognize `*_js.go` files?

**Review by hajimehoshi — COMMENTED** [review]
Hmm, I don't want to have special hacks for GopherJS. Could GopherJS have a compatibility with Wasm APIs (`syscall/js`) and so on?

**Comment by visualfc** [comment]
> Hmm, I don't want to have special hacks for GopherJS. Could GopherJS have a compatibility with Wasm APIs (`syscall/js`) and so on?

gopherjs unsupport unsafe  sliceToByteSlice convert

**Comment by hajimehoshi** [comment]
> gopherjs unsupport unsafe sliceToByteSlice convert

I don't think this is what Ebiten should fix. If it'd be possible to fix GopherJS, it'd be the best.

**Comment by visualfc** [comment]
> > gopherjs unsupport unsafe sliceToByteSlice convert
> 
> I don't think this is what Ebiten should fix. If it'd be possible to fix GopherJS, it'd be the best.

yes, if fixed for gopherjs is best.

**Pull request closed without merging**